dotnet-test-samples/sqs-lambda/docs/README.md

```diff
@@ -0,0 +1,204 @@
+## Amazon Simple Queue Service (SQS) & AWS Lambda Function
+
+### Description
+
+This pattern creates an AWS Lambda function that consumes messages from an Amazon Simple Queue Service (Amazon SQS) queue using SAM and .NET 6.
+
+> **Important:** *This application uses various AWS services and there are costs associated with these services after the Free Tier usage. Please see the AWS Pricing page for details. You are responsible for any AWS costs incurred.*
+
+## Language
+.NET 6
+
+## Framework
+The framework used to deploy the infrastructure is [SAM](https://aws.amazon.com/serverless/sam)
+
+## Services used
+The AWS services used in this pattern are
+
+*Amazon SQS  &rarr;  AWS Lambda*
+
+## Topology
+
+<img src="./image.png" alt="topology" width="80%"/>
+
+## Description
+The SAM template contains all the information to deploy AWS resources (An Amazon SQS queue and an AWS Lambda function) and also the permissions required by these services to communicate.
+
+You will be able to create and delete the CloudFormation stack using the SAM CLI.
+
+## Project Structure
+
+The serverless application was created with an AWS SAM Template.
+
+```
+sam init -r dotnet6
+```
+
+The solution is split down into two projects:
+
+- Function project(s):
+  - [SqsEventHandler](../src/SqsEventHandler/SqsEventHandler.csproj)
+
+- Test project(s):
+  - [SqsEventHandler.UnitTests](../tests/SqsEventHandler.UnitTests/SqsEventHandler.UnitTests.csproj)
+  - [SqsEventHandler.IntegrationTests](../tests/SqsEventHandler.IntegrationTests/SqsEventHandler.IntegrationTests.csproj)
+
+## Deployment commands
+
+The AWS SAM CLI is used to deploy the application. When working through the `sam deploy --guided` take note of the stack name used.
+
+```
+sam init
```

Comment:
This isn't required, just sam build/deploy would be enough to re-use the sample

Reply:
removed